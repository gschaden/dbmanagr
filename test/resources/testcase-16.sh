skiline@localhost/skiline/system_property/HOST
