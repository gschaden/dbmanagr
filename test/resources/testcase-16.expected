Id	Title	Subtitle	Autocomplete	
37e593c2-b7fb-3157-a6ee-dc9b93cb2783	10.60.4.113	HOST_NAME	skiline@localhost/skiline/system_property/1/

